aries-site/src/layouts/main/Header.js

```diff
@@ -48,6 +48,13 @@ const StyledHeader = ({ ...rest }) => {
           onClick={() => setShowSearch(true)}
         />
         {showSearch && <Search setOpen={setShowSearch} />}
+        <Button
+          tip="File an issue or request"
+          icon={<CircleQuestion />}
```

Comment:
hmm wonder if github icon would make more sense

Reply:
Thanks for feedback! -- I actually misunderstood a proposal Oliver had made so this can be ignored for now.

Still an earlier exploration.

Reply:
aw gotcha 👍